app/src/shared/components/ImageUpload/index.jsx

```diff
@@ -101,7 +97,7 @@ const ImageUpload = ({
             {...getRootProps({
                 className: cx(styles.root, styles.ImageUpload, {
                     [styles.dropzoneAdviceImageLoading]: !!uploading,
-                    [styles.imageUploaded]: !uploading && (!!srcImage || uploaded),
```

Comment:
Fixes an issue in editor where you upload an image but decide to click cancel while cropping -> empty white image was shown instead of placeholder.